src/api/api.service.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { ZaLaResponse } from 'src/common/helpers/response';
-import { Repository } from 'typeorm';
+import { Connection, EntityManager, Repository } from 'typeorm';
```

Comment:
If you don't need this anymore, remove it

Reply:
Removed